backend/core/settings.py

```diff
@@ -50,7 +38,7 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 ]
 
-ROOT_URLCONF = 'badgeyay.urls'
```

Comment:
Revert project name to badgeyay

Reply:
But that would affect the whole folder structure and would require many changes. How does the name bother us here?

Reply:
Yes, it was badgeyay previously. Why change it to core?

> But that would affect the whole folder structure and would require many changes

On the contrary. the changes in this PR have affected the whole folder structure and made many changes

Reply:
I am sorry that I don't know how to rename a Django project afterwards. I had made this whole folder structure again because there were some errors in the existing v2 structure where Abstract User usage was restricting me from using the Django's custom authentication.

Reply:
And does this folder name actually bother us? This is not going to affect things later since we are still starting from scratch. Correct me if I am wrong. @iamareebjamal.

Reply:
We already started from scratch in v2 branch

Let's say 2 weeks later someone else comes and faces some issue in the auth module. What should the person do?

1. Fix the module
2. Re-initialize the django project

Reply:
Okay, I am sorry, can you guide me on how to rename this root folder?

Reply:
This won't happen again, however, I would feel grateful if you can just accept it this time.

Reply:
It won't happen again by you, but someone else might. We can't reinitialize the project every 3 days, sorry

Reply:
[New PR](https://github.com/fossasia/badgeyay/pull/2184/commits/e6296b79723f52004db358afd2a4ae4cc45a17fd) fixed everything as you asked for. Now, tell me other changes.